backend/src/main/java/com/backjoongwon/cvi/config/WebMvcConfig.java

```diff
@@ -0,0 +1,38 @@
+package com.backjoongwon.cvi.config;
+
+import com.backjoongwon.cvi.user.auth.AuthenticationPrincipalArgumentResolver;
+import com.backjoongwon.cvi.user.auth.SigninInterceptor;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@Configuration
+public class WebMvcConfig implements WebMvcConfigurer {
+
+    private final SigninInterceptor signinInterceptor;
+    private final AuthenticationPrincipalArgumentResolver resolver;
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**").allowedMethods("*").allowedOriginPatterns("*");
+    }
```

Comment:
```suggestion
    @Override
    public void addCorsMappings(CorsRegistry registry) {
        registry.addMapping("/**")
                .allowedMethods("*")
                .exposedHeaders(HttpHeaders.LOCATION);
    }
```
exposed 헤더 추가 해줘야할 거같아요